### ISSUE TITLE
sdcadm binder (Zookeeper) high availability setup initialization throws exception

### ISSUE THREAD

**Issue by siepkes** [TritonDataCenter/sdcadm#98]
I tried to clusterize Zookeeper with the `sdcadm post-setup ha-binder` command as documented but ran in to the exception posted below. All nodes (including the headnode) were running platform `20220310T212952Z`.

```
# sdcadm post-setup ha-binder headnode compute-nl1-02 compute-nl1-03
Getting SDC's binder details from SAPI
Getting SDC's binder instances from SAPI
Getting SDC's binder vms from VMAPI
Calculating next binder instance alias
Getting SDC's moray details from SAPI
Getting SDC's moray vms from VMAPI
Getting SDC's manatee details from SAPI
Getting SDC's manatees vms from VMAPI
Getting manatee shard status

This command will make the following changes:

Add instance "binder1" in server 00000000-0000-0000-0000-0cc47a4d1ecc
Add instance "binder2" in server 0066a868-5bf4-eb11-8000-3cecef7aa2a6
Update core VMs resolvers.

Would you like to continue? [y/N] y

Freezing manatee shard
Create work dir: /var/sdcadm/ha-binder/20220512T210534Z
Disabling zookeeper for data backup
Creating backup of zookeeper data directory (this may take some time)
Uncaught AssertionError: options.message (string) is required

FROM
_toss (/opt/smartdc/sdcadm/node_modules/assert-plus/assert.js:22:5)
Function.out.(anonymous function) [as string] (/opt/smartdc/sdcadm/node_modules/assert-plus/assert.js:122:17)
new InternalError (/opt/smartdc/sdcadm/lib/errors.js:79:12)
execRemoteCb (/opt/smartdc/sdcadm/lib/steps/zookeeper.js:88:16)
/opt/smartdc/sdcadm/lib/common.js:1418:9
/opt/smartdc/sdcadm/lib/common.js:618:13
ChildProcess.exithandler (child_process.js:190:7)
emitTwo (events.js:106:13)
ChildProcess.emit (events.js:191:7)
maybeClose (internal/child_process.js:920:16)
Process.ChildProcess._handle.onexit (internal/child_process.js:230:5)
/opt/smartdc/bin/sdcadm: line 18: 73134 Illegal Instruction     (core dumped) /opt/smartdc/sdcadm/node/bin/node --abort_on_uncaught_exception /opt/smartdc/sdcadm/lib/cli/index.js "$@"
```

**Comment by bahamat** [comment]
This has been fixed in TritonDataCenter/binder#24. Update binder to latest in the dev channel to get the fix.

**Comment by siepkes** [comment]
Thanks!
